### ISSUE TITLE
Event update form doesn't fully support test events

### ISSUE THREAD

**Issue by melinath** [littleweaver/django-brambling#329]
There are two issues:

1. The test api fields are not excluded from the form, which means they get blanked every time the form saves.
2. If the "disconnect" boxes *are* checked, it always manually blanks the non-test fields. Which means that they'll get erased when they shouldn't, and that if the test fields *weren't* getting blanked, they never could be.

While we're fixing this, we should also implement *actual* oauth disconnects, rather than just deleting our copy of their data.

Original report was in #320.

**Comment by melinath** [comment]
Made a separate ticket for oauth disconnects (#330)

**Comment by melinath** [comment]
This is just waiting on a pull request to be merged.
